### PR TITLE
Refresh network-fortinet-fortiauthenticator-snmp

### PR DESCRIPTION
**Pull request by lucie-dubrunfaut**
## Description

Refresh network-fortinet-fortiauthenticator-snmp

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-fortinet-fortiauthenticator-snmp.md [hunk 1]
```diff
@@ -5,80 +5,127 @@ title: Fortinet FortiAuthenticator SNMP
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
+## Dépendances du Connecteur de supervision
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-fortinet-fortiauthenticator-snmp.md:8]:
```suggestion
## Dépendances du connecteur de supervision
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-fortinet-fortiauthenticator-snmp.md [hunk 2]
```diff
@@ -5,80 +5,127 @@ title: Fortinet FortiAuthenticator SNMP
...
-### Métriques collectées
+<Tabs groupId="sync">
+<TabItem value="Net-Fortinet-Fortiauthenticator-SNMP-custom" label="Net-Fortinet-Fortiauthenticator-SNMP-custom">
+
+| Alias         | Modèle de service                                         | Description                                  |
+|:--------------|:----------------------------------------------------------|:---------------------------------------------|
+| Authenticator | Net-Fortinet-Fortiauthenticator-Authenticator-SNMP-custom | Contrôle les statistiques d'authentification |
+| Cpu           | Net-Fortinet-Fortiauthenticator-Cpu-SNMP-custom           | Contrôle du taux d'utilisation CPU           |
+| Disk-Log      | Net-Fortinet-Fortiauthenticator-Disk-Log-SNMP-custom      | Contrôle l'utilisation du disque de log      |
+| Ha            | Net-Fortinet-Fortiauthenticator-Ha-SNMP-custom            | Contrôle le statut de la haute-disponibilité |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-fortinet-fortiauthenticator-snmp.md:33]:
```suggestion
| Ha            | Net-Fortinet-Fortiauthenticator-Ha-SNMP-custom            | Contrôle le statut de la haute disponibilité |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-fortinet-fortiauthenticator-snmp.md [hunk 3]
```diff
@@ -5,80 +5,127 @@ title: Fortinet FortiAuthenticator SNMP
...
 
-### Métriques collectées
+<Tabs groupId="sync">
+<TabItem value="Net-Fortinet-Fortiauthenticator-SNMP-custom" label="Net-Fortinet-Fortiauthenticator-SNMP-custom">
+
+| Alias         | Modèle de service                                         | Description                                  |
+|:--------------|:----------------------------------------------------------|:---------------------------------------------|
+| Authenticator | Net-Fortinet-Fortiauthenticator-Authenticator-SNMP-custom | Contrôle les statistiques d'authentification |
+| Cpu           | Net-Fortinet-Fortiauthenticator-Cpu-SNMP-custom           | Contrôle du taux d'utilisation CPU           |
+| Disk-Log      | Net-Fortinet-Fortiauthenticator-Disk-Log-SNMP-custom      | Contrôle l'utilisation du disque de log      |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-fortinet-fortiauthenticator-snmp.md:32]: c'est quoi un disque de log?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-fortinet-fortiauthenticator-snmp.md [hunk 4]
```diff
@@ -5,80 +5,127 @@ title: Fortinet FortiAuthenticator SNMP
...
 
-| Metric name             | Description                               | Unit  |
-| :---------------------  | :---------------------------------------- | :---- |
-| memory.usage.percentage | Percentage of memory usage on the device. |   %   |
+| Nom                     | Unité |
+|:------------------------|:------|
+| memory.usage.percentage | %     |
 
 </TabItem>
 </Tabs>
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-fortinet-fortiauthenticator-snmp.md:131]: agent?

File: pp/integrations/plugin-packs/procedures/network-fortinet-fortiauthenticator-snmp.md [hunk 5]
```diff
@@ -5,77 +5,126 @@ title: Fortinet FortiAuthenticator SNMP
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
+## Connector Dependencies
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-fortinet-fortiauthenticator-snmp.md:8]:
```suggestion
## Connector dependencies
```

File: pp/integrations/plugin-packs/procedures/network-fortinet-fortiauthenticator-snmp.md [hunk 6]
```diff
@@ -5,77 +5,126 @@ title: Fortinet FortiAuthenticator SNMP
...
+The connector brings the following service templates (sorted by the host template they are attached to):
 
-### Collected metrics
+<Tabs groupId="sync">
+<TabItem value="Net-Fortinet-Fortiauthenticator-SNMP-custom" label="Net-Fortinet-Fortiauthenticator-SNMP-custom">
+
+| Service Alias | Service Template                                          | Service Description            |
+|:--------------|:----------------------------------------------------------|:-------------------------------|
+| Authenticator | Net-Fortinet-Fortiauthenticator-Authenticator-SNMP-custom | Check authenticator statistics |
+| Cpu           | Net-Fortinet-Fortiauthenticator-Cpu-SNMP-custom           | Check cpu usage                |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-fortinet-fortiauthenticator-snmp.md:30]:
```suggestion
| Cpu           | Net-Fortinet-Fortiauthenticator-Cpu-SNMP-custom           | Check CPU usage                |
```

File: pp/integrations/plugin-packs/procedures/network-fortinet-fortiauthenticator-snmp.md [hunk 7]
```diff
@@ -5,77 +5,126 @@ title: Fortinet FortiAuthenticator SNMP
...
+| Disk-Log      | Net-Fortinet-Fortiauthenticator-Disk-Log-SNMP-custom      | Check log disk usage           |
+| Ha            | Net-Fortinet-Fortiauthenticator-Ha-SNMP-custom            | Check high-availability status |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-fortinet-fortiauthenticator-snmp.md:32]:
```suggestion
| Ha            | Net-Fortinet-Fortiauthenticator-Ha-SNMP-custom            | Check high availability status |
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-fortinet-fortiauthenticator-snmp.md [hunk 8]
```diff
@@ -5,202 +5,588 @@ title: Fortinet FortiAuthenticator SNMP
...
 
-### Métriques collectées
+<Tabs groupId="sync">
+<TabItem value="Net-Fortinet-Fortiauthenticator-SNMP-custom" label="Net-Fortinet-Fortiauthenticator-SNMP-custom">
+
+| Alias         | Modèle de service                                         | Description                                  |
+|:--------------|:----------------------------------------------------------|:---------------------------------------------|
+| Authenticator | Net-Fortinet-Fortiauthenticator-Authenticator-SNMP-custom | Contrôle les statistiques d'authentification |
+| Cpu           | Net-Fortinet-Fortiauthenticator-Cpu-SNMP-custom           | Contrôle du taux d'utilisation CPU           |
+| Disk-Log      | Net-Fortinet-Fortiauthenticator-Disk-Log-SNMP-custom      | Contrôle l'utilisation du disque de log      |
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-fortinet-fortiauthenticator-snmp.md:32]:
```suggestion
| Disk-Log      | Net-Fortinet-Fortiauthenticator-Disk-Log-SNMP-custom      | Vérifier l'utilisation du disque d'enregistrement      |
```

**Review by cg-tw — APPROVED** [review]

**Pull request merged**